watchdog.c

```diff
@@ -99,6 +110,7 @@ start(lua_State *L)
 static int
 stop(lua_State *L)
 {
+	(void)L;
```

Comment:
That's an interesting place. What if we save it to a static variable? It could help with https://github.com/tarantool/watchdog/issues/10#issuecomment-566538007

Reply:
I don't understand how it could help in "stop" function